packages/lib/src/types/alluvial.ts

```diff
@@ -5,9 +5,9 @@ import {
   SankeyNode as D3SankeyNode,
   SankeyExtraProperties,
 } from 'd3-sankey';
```

Comment:
This could be imported as `import type` and there doesn't seem to be any issue. But skipping it because of the following concern mentioned in TS release notes.

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html
<img width="920" alt="Screenshot 2024-04-03 at 11 45 55" src="https://github.com/Adyen/lume/assets/8277926/bd200d29-5236-4146-ae8d-2d64d16d34a4">